apps/api/src/app/messages/usecases/get-messages/get-messages.command.ts

```diff
@@ -1,18 +1,22 @@
 import { ChannelTypeEnum } from '@novu/shared';
-import { IsNumber, IsOptional } from 'class-validator';
+import { IsArray, IsNumber, IsOptional, IsString } from 'class-validator';
 import { EnvironmentCommand } from '../../../shared/commands/project.command';
 
 export class GetMessagesCommand extends EnvironmentCommand {
-  @IsNumber()
-  @IsOptional()
-  page: number;
-
   @IsOptional()
   subscriberId?: string;
 
   @IsOptional()
   channel?: ChannelTypeEnum;
 
   @IsNumber()
-  limit: number;
+  page = 0;
+
+  @IsNumber()
+  limit = 10;
+
+  @IsOptional()
+  @IsArray()
+  @IsString({ each: true })
+  transactionId?: string[] | null;
```

Comment:
I missed this. Considering that in the controller we always check it is an array or we build an array of one transactionId I think we should rename this to `transactionIds`. Also I'd rather have the fallback to `undefined` rather than `null`.
```suggestion
  transactionIds?: string[] | null;
```

Reply:
Done ✅